roles/ood_user_reg_cloud/templates/ini.j2

```diff
@@ -1,7 +1,6 @@
 [uwsgi]
 mount = /register=wsgi.py
 
-uid = {{ RegUser_app_user }}
```

Comment:
This change is good.  But I need to withhold this PR until the merge of the gunicorn feature since the ini file will change completely.